src/Proto.Actor/IContext.cs

```diff
@@ -113,6 +113,15 @@ public interface IContext : ISenderContext
         /// <param name="target">the Task to await</param>
         /// <param name="action">the continuation to call once the task is completed</param>
         /// <typeparam name="T">The generic type of the task</typeparam>
-        void ReenterAfter<T>(Task<T> target, Func<Task<T>,Task> action);
+        void ReenterAfter<T>(Task<T> target, Func<Task<T>, Task> action);
```

Comment:
Instead of having to return a task maybe it could look like this?

`void ReenterAfter<T>(Task<T> target, Action<Task<T>> action);`